openfe/tests/setup/test_network_planning.py

```diff
@@ -133,6 +138,29 @@ def scoring_func(mapping):
             assert 'score' not in edge.annotations
 
 
+@pytest.mark.parametrize('multi_mappers', [False, True])
+def test_minimal_spanning_network_mappers(atom_mapping_basic_test_files, multi_mappers):
+    ligands = [atom_mapping_basic_test_files['toluene'],
+               atom_mapping_basic_test_files['2-naftanol'],
+               ]
+
+    if multi_mappers:
+        mappers = [BadMapper(), openfe.setup.atom_mapping.LomapAtomMapper()]
+    else:
+        mappers = openfe.setup.atom_mapping.LomapAtomMapper()
+
+    def scorer(mapping):
+        return 1.0 / len(mapping.componentA_to_componentB)
+
+    network = openfe.ligand_network_planning.generate_minimal_spanning_network(
+        ligands=ligands,
+        mappers=mappers,
+        scorer=scorer,
+    )
+
+    assert isinstance(network, openfe.LigandNetwork)
```

Comment:
Add an assertion on the number of edges? (can be a `MultiDiGraph`, but shouldn't actually be multi).